### DISCUSSION TITLE
v-date-picker documentation in Vuetify 3

### DISCUSSION THREAD

**Discussion by benatlas** [vuetifyjs/vuetify#16988] [Q&A] [answered]
It seems there is no documentation on the v-date-picker in version 3 of the documentation? I have searched for v-date-picker in the search but no result (same method in version 2 works). I have looked under components > pickers, does not appear there either. Is there somewhere I can look for documentation on this component in version 3?

Thanks

**Comment by simenkf** [comment, marked as answer]
Not everything is upgraded to Vuetify 3 yet. If you look at the labs page (https://vuetifyjs.com/en/labs/introduction/) you'll see that it is planned to be available there in in Q2 2023. Components in labs are work in progress you can start using so that you are ready once the component is officially released (prod ready).
